doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc

```diff
@@ -54,6 +54,14 @@ In this command line:
 ** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URL for {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that the {KIE_SERVER} is to join. If you are deploying several {KIE_SERVERS}, you must ensure each of the servers has a different application name.
 ** *KIE Server Certificate Name* (`KIE_SERVER_HTTPS_NAME`): The name of the certificate in the keystore that you created in <<secrets-kie-create-proc>>.
 ** *KIE Server Keystore Password* (`KIE_SERVER_HTTPS_PASSWORD`): The password for the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server Mode* (`KIE_SERVER_MODE`): Set to `DEVELOPMENT` if you need to deploy a `SNAPSHOT` version of a KJAR artifact. 
```

Comment:
The other list items contain a description, not just how to use them. You may want to add a brief description.